disperser/dataapi/subgraph_client.go

```diff
@@ -274,6 +274,22 @@ func (sc *subgraphClient) QueryIndexedOperatorsWithStateForTimeWindow(ctx contex
 	}, nil
 }
 
+func (sc *subgraphClient) QueryRegisteredOperators(ctx context.Context) (*IndexedQueriedOperatorInfo, error) {
```

Comment:
The `QueryIndexedRegisteredOperatorsForTimeWindow` may already support this by passing right parameter value

Reply:
Removing